.github/workflows/python-test.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
           run: black . --check --verbose
         - name: Run Python tests
           if: runner.os != 'Windows'
-          run: python3 -m pytest -s -vv --cov --cov-fail-under=87
+          run: python3 -m pytest -s -vv --cov --cov-fail-under=86
```

Comment:
We fixed a test? :)

Reply:
or we increased code coverage

Reply:
Sigh I had upped this to 87 in a branch this week as it seemed we were reaching this, but one of the merged changes must have decreased it, as our nightly builds are also now failing. So I had to reduce it :(